client/signup/jetpack-connect/plans.jsx

```diff
@@ -73,7 +106,7 @@ const Plans = React.createClass( {
 
 		return (
 			<div>
-				<Main>
+				<Main isWideLayout={ true }>
```

Comment:
My apologies, but the prop is called `wideLayout`, not `isWideLayout` as I mistakenly said before. Corrected in d34c50a.